openapi-v1.yaml

```diff
@@ -3461,7 +3684,7 @@ definitions:
         type: string
         x-nullable: true
       value:
-        $ref: "#/definitions/TGArgValue"
+        $ref: '#/definitions/TGArgValue'
 
   TGArgValue:
```

Comment:
`TGArgValue` has no properties, only description. Is this intended?

Reply:
Yes, this is intended. A TGArgValue can be any JSON object; I want to include at least a short description of the format in the API spec itself. (I’m also going to write a more precise, longer-form description elsewhere.)